webpack/assets/javascripts/react_app/common/I18n/intl.js

```diff
@@ -0,0 +1,7 @@
+const htmlElement = document.getElementsByTagName('html')[0];
+
+export default {
+  locales: window.locales || {},
+  locale: htmlElement.getAttribute('lang') || 'en',
+  timezone: htmlElement.getAttribute('data-timezone') || 'UTC',
```

Comment:
What happens if we set "Browser TZ/lang" wouldn't that fall back into 'en'/'UTC'?

Reply:
It gets set and works as charm :+1: